backend-services/user-verification-service/src/main/resources/application.properties

```diff
@@ -50,12 +50,13 @@ spring.profiles.active=${SPRING_PROFILES_ACTIVE:dev--preprod}
 cors.allowed.origins=${CORS_ALLOWED_ORIGINS:http://localhost:3000}
 
 ledger.follower.app.base.url=${LEDGER_FOLLOWER_APP_URL:http://localhost:9090}
+keri.ballot.verifier.base.url=${KERI_BALLOT_VERIFIER_APP_URL:http://localhost:5667}
 
 aws.sns.accessKeyId=${AWS_SNS_ACCESS_KEY_ID}
 aws.sns.secretAccessKey=${AWS_SNS_SECRET_ACCESS_KEY}
 aws.sns.region=${AWS_SNS_REGION:eu-west-1}
 
-friendly.custom.name=${FRIENDLY_CUSTOM_NAME:Cardano Summit 2023}
+friendly.custom.name=${FRIENDLY_CUSTOM_NAME:CF04}
```

Comment:
Is CF04 friendly custom name :)

Reply:
No, I used these acronyms during my tests using public(free use) phone numbers. I just updated it to `Cardano Summit 2024`